sspl_test/rabbitmq/rabbitmq_ingress_processor_tests.py

```diff
@@ -228,6 +228,13 @@ def _configure_exchange(self):
             self._password = self._conf_reader._get_value_with_default(
                 self.RABBITMQPROCESSORTEST, self.PASSWORD, "sspl4ever"
             )
+            self.cluster_id = self._conf_reader._get_value_with_default(

```

Comment:
this is where sspl sanity were still failing seems like ,even when Mohit has addressed earlier

Reply:
Yes, this was the issue! Thanks @Madhura-08

Reply:
No problem Mohit!